shared/js/option_menu.js

```diff
@@ -169,10 +169,13 @@ var OptionMenu = function(options) {
     if (typeof action !== 'undefined') {
       method = action.method || function() {};
 
-      method.apply(null, action.params || []);
+      method(...(action.params || []));
 
-      // Hide action menu when click is received
-      this.hide();
+      // Hide action menu automatically when hideOnAction does not set to true.
+      if (!action.hideOnAction) {
+        console.log('@@@ hide');
```

Comment:
__reminder:__ don't forget to remove this ```console.log``` before landing.

__question:__ mm the name ```hideOnAction``` and what we do in reality is a bit confusing :) Should not it be like this?:
```
{ 
	l10nId: `replace-attachment-${fileType}`,
	method: onReplace,
	hideOnAction: false
}
.....

var hideOnAction = typeof action.hideOnAction === 'boolean' ? action.hideOnAction : true;
if (hideOnAction) {
  hide();
}
```